Cargo.toml

```diff
@@ -15,4 +15,6 @@ vob = { version="2.0", features=["serde"] }
 packedvec = { version="1.0", features=["serde"] }
 serde = { version="1.0", features=["derive"], optional=true }
 num-traits = "0.2"
+
+[dev-dependencies]
```

Comment:
Maybe I'm an idiot, but `rand` wasn't previously a direct dependency of this crate, so what does this change?

Reply:
But it was. Only the `[dev-depencies]` bit is new. `rand = 0.5` has been there all along.

Reply:
I am an idiot!

Reply:
What this changes is that `rand` is only build when running `cargo test`. When running `cargo build` (e.g. when using this crate in another project) then `rand` isn't built.